yb-voyager/cmd/root.go

```diff
@@ -291,6 +291,21 @@ var noPersistentPreRunNeededList = []string{
 	"yb-voyager end",
 }
 
+// used for registering the --config-file flag
+var offlineCommands = []string{
+	"yb-voyager assess-migration",
+	"yb-voyager export schema",
+	"yb-voyager analyze-schema",
+	"yb-voyager import schema",
+	"yb-voyager export data",
+	"yb-voyager export data from source",
+	"yb-voyager import data",
+	"yb-voyager import data to target",
+	"yb-voyager finalize-schema-post-data-import",
+	"yb-voyager end migration",
+	"yb-voyager compare-performance",
+}
```

Comment:
Not related to the PR.
But we support Live migration workflows also with a config file, so we should add this flag to help in all commands cc: @ShivanshGahlot @shubham-yb